blocks/carousel/carousel.css

```diff
@@ -114,7 +117,6 @@
     flex-wrap: nowrap;
 }
 
-.carousel.image-carousel-full-width figure,
```

Comment:
Please revert this deleted line -- We still need to hide this when the other 'full width' variation of the carousel is use.